docs/load-balancer/sticky.md

```diff
@@ -0,0 +1,29 @@
+# Sticky Load Balancing
+
+The sticky load balancer selects a single service instance and keeps using it until it fails.
+Then, it selects the next one.
+
+It is possible to configure a backoff time to specify for how long a failing service instance should not be retried.
+
+Precisely, the load balancer works as follows:
+
+* if the previously selected service instance has not failed, and is still available, return it;
+* else if exists, return the first available service instance that has no recorded failure;
+* else, find the available instance for which the time since the last failure is the greatest, and
```

Comment:
```suggestion
* else, find an available instance for which the time since the last failure is the longest, and
```

Reply:
Why `an` and not `the`? We're talking about a specific instance here.